### DISCUSSION TITLE
Using `styled()` with `Animated.View`

### DISCUSSION THREAD

**Discussion by JWesorick** [nativewind/nativewind#411] [Q&A] [answered]
I'm trying to use `styled(Animated.View)` but they don't seem to mix well.

Ex. this...
```jsx
const AnimatedView = styled(Animated.View);


<AnimatedView
  className="absolute rounded-full"
  style={{
    left: offsetDot,
  }}
  {...panResponder.panHandlers}
/>
```

will throw an error `styleq: left typeof undefined is not "string".` How can I style `Animated.View with tailwind classes and still have animated `style` values?

**Comment by marklawlor** [comment, marked as answer]
React Native Web 0.18 has a known bug when using classNames with Animated components. You will need to update to the RNW 0.19 preview release https://github.com/necolas/react-native-web/pull/2377

**Comment by danstepanov** [comment]
Closing this discussion, if this is still relevant, feel free to open a new one.
